docs/user_endpoints.md

```diff
@@ -217,3 +218,28 @@ Status: 200
   ]
 }
 ```
+
+## GET /users/members/me
+Request
+```ruby
+  GET /users/members/me
+```
+
+Response
+```ruby
+Status: 200
+{
+  "id": 1,
+  "name": "Firstname Lastname",
+  "points": 10,
+  "image": "<img_url>",
```

Comment:
please in the docs, give an example, shorthand examples like these may be confusing! One should be able to know it's a string, object, array, integer, date!